### DISCUSSION TITLE
Pandoc ignores custom template?

### DISCUSSION THREAD

**Discussion by johnmapeson** [jgm/pandoc#9786] [Q&A] [answered]
I try to convert from Markdown to HTML using a custom template. I copied my custom `default.html5` to the user data directory, `~/.local/share/pandoc` (as shown by `pandoc --version`), and executed `pandoc -s input.md -o output.htm`.

My custom `default.html5` is simply a copy of https://raw.githubusercontent.com/jgm/pandoc-templates/master/default.html5 with the `<title>` element deleted.

``` html
<!-- this line is deleted -->
<title>$if(title-prefix)$$title-prefix$ – $endif$$pagetitle$</title>
```

When I execute `pandoc -s input.md -o output.htm`, Pandoc seems to ignore that custom `default.html5`, because when I examine the resulting HTML files, the `<title>` element is still exist. What I'm doing wrong?

pandoc 3.1.13

**Comment by jpcirrus** [comment, marked as answer]
It needs to be in the `templates` subdirectory, which you likely have to create.

**Reply by johnmapeson** [reply]
Many thanks, John, I completely forgot about it.

**Reply by johnmapeson** [reply]
Weird! It worked few times, and now it doesn't work. Seems to be because I haven't sleep, but nevertheless really frustrating.

`default.html5` is in `~/.local/share/pandoc/templates`

The command to convert is `pandoc input.md -s -o output.htm`

update: Nevermind, see https://github.com/jgm/pandoc/issues/9789
